README.md

```diff
@@ -9,7 +9,7 @@ Configs matching a given schema are sourced from the following locations in the
 - Command line argument set in the schema via `commandLineArg`.
 - Environment variable set in the schema via `envVar`.
 - Deployment config file declared via:
-  - Command line argument: `--deploy-config` or `--dc`.
+  - Command line argument: `--deploy-config` or `--df`.
```

Comment:
I'd say either do

1) `--deploy-config` and `--dc`
2) `--deploy-file` and `--df`

I'd stick with one style or the other, not a mixture of the two. As well make sure it follows in `getDeployConfig.js`

Reply:
My vote is 2 and this will help us keep an existing API in apps that will now use this

Reply:
```suggestion
  - Command line argument: `--deploy-config-file` or `--df`.
```

Reply:
I believe this will need a code change as well

Reply:
Yes, but if we're gonna go with `--df` it needs to match the longer version. Otherwise `--dc` makes more sense.

Reply:
I like your suggestion of `--deploy-config-file` or `--df`.

@lannonbr what are your thoughts

Reply:
I'm good with that

Reply:
Alright thanks I'll make the changes